src/main/java/no/unit/nva/model/File.java

```diff
@@ -66,28 +73,34 @@ public void setLicense(License license) {
         this.license = license;
     }
 
-    @Override
-    public boolean equals(Object o) {
```

Comment:
I think deep equals in the data-model is necessary. We should bear in mind that we will potentially need to compare equivalent instances and we will need deep equals.